VocaDbWeb/Startup.cs

```diff
@@ -119,6 +119,33 @@ public void ConfigureServices(IServiceCollection services)
 				});
 
 			services.AddLaravelMix();
+
+			services.AddCors(options =>
+			{
+				options.AddDefaultPolicy(builder =>
+				{
+					builder
+						.AllowAnyOrigin()
+						.AllowAnyHeader()
+						.WithMethods("GET");
+				});
+
+				options.AddPolicy(AuthenticationConstants.AuthenticatedCorsApiPolicy, builder =>
+				{
+					builder
+						.AllowAnyHeader()
+						.WithMethods("GET", "POST", "PUT", "OPTIONS");
+
+					var origins = AppConfig.AllowedCorsOrigins;
+					if (!string.IsNullOrEmpty(origins))
+					{
+						if (origins != "*")
+							builder.WithOrigins(origins.Split(','));
+						else
+							builder.AllowAnyOrigin();

```

Comment:
Actually, for safety reasons, I would leave this out (I am aware that previously it allowed any origin if no origins are configured, but that can be really unsafe).

Also, I think that credentials (with AllowCredentials) should be enabled, because some services use the authentication cookie, such as the [UTAU lyrics wiki importer](https://github.com/VocaDB/vocadb-utau-lyrics-wiki-importer), and AllowAnyOrigin cannot be used together with AllowCredentials.

Reply:
Was [UTAU Lyrics Wiki importer](http://vocaloid.eu/vocadb/UtauLyricsWikiImporter/) already broken? http://vocaloid.eu/utaulyricswiki/api/lyrics-wiki?pagename= throws an internal server error.

Reply:
I don't know. I haven't used it for a while. Not sure if it's actually being used. But the point is that services like that should be supported in some way.